core/src/test/resources/HistoryServerExpectations/limit_app_list_json_expectation.json

```diff
@@ -28,19 +43,4 @@
   } ],
   "id" : "application_1628109047826_1317105",
   "name" : "Spark shell"
-}, {
-  "attempts" : [ {
```

Comment:
This entry is removed when generating the golden file as the number of entries is expected to be three.
https://github.com/apache/spark/blob/eec092c9f9d1ad3df0390b1ea56340c1103ba887/core/src/test/scala/org/apache/spark/deploy/history/HistoryServerSuite.scala#L140